scripts/pg2sphinx_trigger.py

```diff
@@ -1,25 +1,26 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
 # -*- coding: utf-8 -*-
 #
-# read sphinx config file and update all indexes related to input database or input table or index pattern
+# read sphinx config file and update all indexes related to input database or
+# input table or index pattern
 
 import os
-import sys
-import time
-import getpass
 import re
 import sqlite3
-import psycopg2
-import optparse
 import subprocess
-from optparse import OptionParser
+import sys
+from optparse import OptionParser  # pylint: disable=deprecated-module
 
+import psycopg2  # pylint: disable=import-error
```

Comment:
psycopg2 should be part of the Pipfile and then you won't have this pylint error

Reply:
it is only running inside the container. do we use virtualenv inside the container too?

Reply:
Ok I see you use the apt `python3-psycopg2` in Dockerfile. Then for local development with IDE like vscode and co I would suggest to add psycopg2 to the Pipfile dev dependencies and remove the pylint error. A comment in pipfile why it is in dev dependencies might be welcome.